app/controllers/collections_controller.rb

```diff
@@ -8,4 +8,26 @@ def form_class
   def presenter_class
     CollectionShowPresenter
   end
+
+  def manifest
+    respond_to do |f|
+      f.json do
+        render json: manifest_builder
+      end
+    end
+  end
+
+  private
+
+    def manifest_builder
+      ManifestBuilder.new(presenter, ssl: request.ssl?, services: AuthManifestBuilder.auth_services(login_url, logout_url))
+    end
+
+    def login_url
+      main_app.user_omniauth_authorize_url(:cas)
```

Comment:
We are hard-coding the auth provider here and elsewhere.  It's beyond the scope of this ticket to fix that, but maybe we should have a new issue to start inspecting the omniauth providers instead?  That would be one thing we'd need to do to make it possible for someone else to use Plum with a different auth system.